src/react/components/pages/projectSettings/projectSettingsPage.scss

```diff
@@ -4,11 +4,17 @@
     height: 100%;
 
     &-settings {
-        flex-grow: 1;
+        flex-grow: 2;
```

Comment:
Why setting this to 2? 1 should be correct. since it is the only element growing within the flexbox.

Reply:
i was trying to give the loading metrics panel more space, but I think min-width is a better approach